vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java

```diff
@@ -40,7 +40,7 @@ public GeneralInformationLayout() {
   }
 
   private void initContent() {
-    Span generalInformationHeader = new Span("General Information");
+    Span generalInformationHeader = new Span("Batch Information");
```

Comment:
Maybe we have to rename the class accordingly? Currently it is named `GeneralInformationLayout` which would change to `BatchInformationLayout`